src/silx/gui/plot/CompareImages.py

```diff
@@ -32,14 +32,14 @@
 import logging
 import numpy
 import math
+from typing import Optional
 
 import silx.image.bilinear
 from silx.gui import qt
 from silx.gui import plot
 from silx.gui.colors import Colormap
 from silx.gui.plot import tools
 from silx.utils.weakref import WeakMethodProxy
```

Comment:
```suggestion
from silx.utils.deprecation import deprecated_warning
from silx.utils.weakref import WeakMethodProxy
```

Reply:
I prefer not. It's pointless, and @payno is agree to drop that.

Reply:
Ok let's try it anyway, i have some time before the rain